Utilities/src/main/java/edu/stanford/hivdb/mutations/MutationSet.java

```diff
@@ -45,7 +43,7 @@
 
 	private static final long serialVersionUID = -1835692164622753L;
```

Comment:
Constant, please uppercase

Reply:
Ooo I see. I misunderstood your comment earlier that linked to the serializable interface doc. Thanks for clarifying.

Reply:
No problem.